features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -316,6 +316,9 @@
             <ScopeValidator class="org.wso2.carbon.identity.oauth2.validators.xacml.XACMLScopeValidator"/>
         </ScopeValidators>
 
+        <!-- Uncomment below to disable scope validation for implicit grant and authorization code grant. -->
```

Comment:
need to add configuration  to the toml file

Reply:
done [https://github.com/wso2/product-is/pull/5817](https://github.com/wso2/product-is/pull/5817)

Reply:
Would be better to document since which version we support this config. Check the identity.xml for a similar description

Reply:
done